flow/e2e/mongo/mongo_test.go

```diff
@@ -760,16 +756,13 @@ func (s MongoClickhouseSuite) Test_Json_Types() {
 		require.Contains(t, row, `"neg_int16":-32768`)
 		require.Contains(t, row, `"neg_int32":-2147483648`)
 		require.Contains(t, row, `"neg_int64":-9223372036854775807`)
-		require.Contains(t, row, `"float32":3.14`)
 		require.Contains(t, row, `"float64":3.14159265359`)
-		require.Contains(t, row, `"neg_float32":-3.14`)
 		require.Contains(t, row, `"neg_float64":-3.14159265359`)
+		require.Contains(t, row, `"max_float64":9223372036854776000`)
+		require.Contains(t, row, `"min_float64":-9223372036854776000`)
```

Comment:
note: this is display issue in clickhouse console, just like javascript

Reply:
This one is coming from `encoding/json.Marshal()`, and Delve debugger does the same :/
Seems like it's just in the imprecise float territory across the implementations, even if the specific value might seem special